src/vsr/replica.zig

```diff
@@ -373,21 +372,23 @@ pub fn ReplicaType(
             while (!self.opened) self.superblock.storage.tick();
             assert(self.superblock.working.vsr_state.internally_consistent());
 
-            const node_count = options.replica_count + options.standby_count;
-            if (self.superblock.working.replica >= node_count) {
-                log.err("{}: open: no address for replica (node_count={})", .{
-                    self.superblock.working.replica,
-                    node_count,
+            const replica = self.superblock.working.vsr_state.replica;
+            const replica_count = self.superblock.working.vsr_state.replica_count;
+            if (replica >= options.node_count or replica_count > options.node_count) {
+                log.err("{}: open: no address for replica (replica_count={} node_count={})", .{
+                    self.superblock.working.vsr_state.replica,
+                    replica_count,
+                    options.node_count,
                 });
                 return error.NoAddress;
```

Comment:
This logic here becomes even more awkward now. It would make sense to pass `standby_cont` rather that `node_count`, but the problem is that that won't work for `start` command: we know node count (length of addresses), but we'll only learn replica_count (and standby_count) after we open the superblock. 

And we can't just read addresses.len here, because not all message busses actually have addresses. 

I think this'll be less illogical after #529